src/components/rating.html

```diff
@@ -0,0 +1,240 @@
+  </head>
+  <body>
+    <div class="ui-sideNav gutter-medium">
+      <div class="ui-sideNav-header text-medium text-color-white">
+        <span class="ui-sideNav-logo gutter-small">C</span>
+        <span class="gutter-medium"
+          ><a
+            href="../../index.html"
+            class="text-color-white text-decoration-none"
+            >UI Docs</a
+          ></span
+        >
+        <i
+          class="fas fa-times-circle ui-sideNav-icon text-medium"
+          aria-hidden="true"
+          onClick="closeSideNav()"
+        ></i>
+      </div>
+      <div class="ui-sideNav-items text-medium text-color-white gutter-medium">
+        <div class="ui-sideNav-item"><h4>Getting Started</h4></div>
+        <div class="ui-sideNav-item">
+          <a
+            href="../../src/usage.html"
+            id="usage-link"
+            class="text-color-white gutter-small text-decoration-none"
+            >Usage</a
+          >
+        </div>
+        <div class="ui-sideNav-item"><h4>Components</h4></div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./avatar.html"
+            id="avatar-link"
+            class="text-color-white gutter-small text-decoration-none"
+            >Avatar</a
+          >
+        </div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./alert.html"
+            id="alert-link"
+            class="text-color-white gutter-small text-decoration-none"
+            >Alert</a
+          >
+        </div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./badge.html"
+            id="badge-link"
+            class="text-color-white gutter-small text-decoration-none"
+            >Badge</a
+          >
+        </div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./button.html"
+            id="button-link"
+            class="text-color-white gutter-small text-decoration-none"
+            >Button</a
+          >
+        </div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./card.html"
+            id="card-link"
+            class="text-color-white gutter-small text-decoration-none"
+            >Card</a
+          >
+        </div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./image.html"
+            id="image-link"
+            class="text-color-white gutter-small text-decoration-none"
+            >Image</a
+          >
+        </div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./input.html"
+            id="input-link"
+            class="text-color-white gutter-small text-decoration-none"
+            >Input</a
+          >
+        </div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./rating.html"
+            class="text-color-white gutter-small text-decoration-none"
+            >Rating</a
+          >
+        </div>
+        <div class="ui__sideNav__item">
+          <a
```

Comment:
i think in this class name you have used double underscores but in rest of the classes you have use single hyphens. same issue in rest of the files